src/it/java/uk/gov/companieshouse/company_appointments/officerappointments/OfficerAppointmentsControllerITest.java

```diff
@@ -61,6 +77,9 @@ static void start() throws IOException {
         mongoTemplate.insert(Document.parse(IOUtils.resourceToString("/appointment-data2.json", StandardCharsets.UTF_8)), DELTA_APPOINTMENTS_COLLECTION);
         mongoTemplate.insert(Document.parse(IOUtils.resourceToString("/appointment-data3.json", StandardCharsets.UTF_8)), DELTA_APPOINTMENTS_COLLECTION);
         System.setProperty("company-metrics-api.endpoint", "localhost");
+
+        corporateOfficerId = UUID.randomUUID().toString();
```

Comment:
These can be assigned on line 58/9 where the variables are declared, and then made `static final`